### DISCUSSION TITLE
Anzeige mobile Geräte (Android) neue Instanz auf Raspi5/Bookworm

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#3593] [Q&A]
### Beschreibung (auf Deutsch oder Englisch)

Ich habe eine neue Instanz auf einem Raspi5 OS Bookworm installiert
Und soweit alles ok, Dank Deiner Hilfe, Georg.
Jetzt fällt mir auf, dass wie schon mal (schon ne Weile her) die Anzeige gestaucht dargestellt wird weil der Browser oben die URL Zeile einblendet
und unten die Systembuttons gezeigt werden.
Über die Einstellung Vollbild anzeigen geht es nur bis zu einem erneuten Aufruf.
Meine erste Instanz von Solectrus, auf einem Raspi4 OS Bullseye wird formatfüllend angezeigt.

Raspi5
![grafik](https://github.com/user-attachments/assets/6efb3a12-0778-4271-aca0-e489da9708eb)

Raspi4
![grafik](https://github.com/user-attachments/assets/64d3b8d6-cfef-465b-8f69-6973eb44dd6d)

Außerdem habe ich in der URL ein "Warndreieck"
![grafik](https://github.com/user-attachments/assets/16c58b62-bc77-44fe-94b4-14e2898134ee)

inhaltlich heißt es da:
![grafik](https://github.com/user-attachments/assets/c6684956-f84e-452a-bad0-b1a8d83026ca)

Die Icons (egal ob Firefox oder Chrome) habe ich genau gleich erstellt wie die des Raspi4, wo es top aussieht.

LG Ernst

### Welche Version von SOLECTRUS verwendest du?

17,1

### Auf welchem Gerät läuft bei dir SOLECTRUS?

Raspberry 5 mit OS Bookworm

**Comment by ledermann** [comment]
Mit dem Raspi selbst dürfte das nichts zu tun haben. Eher mit der verwendeten Version - ist es in beiden Fällen die aktuelle 0.17.1? Und verwendest du in beiden Fällen das gleiche Android-Gerät und den gleichen Browser?

Ansonsten: Hast du das hier berücksichtigt?
https://docs.solectrus.de/bedienung/smartphone

Das Warndreieck ist normal, bei lokalen Servern gibt es kein https. Das kann ignoriert werden.

**Comment by Zeppelin-PV** [comment]
für beide Instanzen Raspi4 und Raspi5 werden die selben Geräte benutzt, Samsung A54 und Lenovo K10
auf beiden stehen Chrome und Firefox zur Verfügung, benutze Chrome, und folge der Beschreibung wie von Dir erwähnt.
auf dem Raspi4 hat das Icon auf dem Bildschirm "Solectrus" als Schreibweise, der Raspi5 "SOLECTRUS"
... aber das wird's nicht sein, ich prüfe nochmals alles auf Unterschiede der beiden Raspi's

**Comment by ledermann** [comment]
Seltsam. Die Ursache würde ich eher beim Browser suchen. Auch dass die Icons unterschiedlich beschriftet sind, ist merkwürdig. Man Icons natürlich umbenennen, die Vorgabe ist aber alles in Großbuchstaben.

Im Zweifelsfall beide Icons nochmal vom Gerät löschen und dann neu anlegen.

**Comment by Zeppelin-PV** [comment]
Du hast recht, Georg
Irgend was hat sich wohl bei den "großen" Browsern geändert.
Hab lange probiert und mit Kiwi einen gefunden der Solectrus so anzeigt wie ich es gewohnt bin.
-- Gewohnheiten sind das Rückgrat des Alltags

**Reply by ledermann** [reply]
Ok, gut, dass es behoben ist. So ganz verstehe ich die Problematik aber nicht, bin in der Android-Welt aber auch nicht zu Hause.
